tests/install/openqa_webui.pm

```diff
@@ -17,16 +17,21 @@ sub install_from_repos {
     $add_repo = "zypper -n ar -p 95 -f obs://devel:openQA/$repo openQA";
     assert_script_run($_) foreach (split /\n/, $add_repo);
     assert_script_run('retry -s 30 -- sh -c "zypper -n --gpg-auto-import-keys ref && zypper --no-cd -n in openQA-local-db"', 600);
-    my $configure = <<'EOF';
-/usr/share/openqa/script/configure-web-proxy
-sed -i -e 's/#.*method.*OpenID.*$/&\nmethod = Fake/' /etc/openqa/openqa.ini
-systemctl restart apache2
-systemctl enable --now openqa-webui
-systemctl status --no-pager openqa-webui
-systemctl enable --now openqa-scheduler
-systemctl status --no-pager openqa-scheduler
-EOF
-    assert_script_run($_) foreach (split /\n/, $configure);
+    if (check_var('OPENQA_WEB_PROXY', 'nginx')) {
+        assert_script_run('retry -s 30 -- sh -c "zypper --no-cd -n in nginx"', 600);
+        assert_script_run "/usr/share/openqa/script/configure-web-proxy --nginx";
+        assert_script_run "systemctl disable --now apache2";
+        assert_script_run "systemctl restart nginx";
+    }
+    else {
+        assert_script_run "/usr/share/openqa/script/configure-web-proxy";
+        assert_script_run "systemctl restart apache2";
+    }
+    assert_script_run q{sed -i -e 's/#.*method.*OpenID.*$/&\nmethod = Fake/' /etc/openqa/openqa.ini};
+    assert_script_run "systemctl enable --now openqa-webui";
+    assert_script_run "systemctl status --no-pager openqa-webui";
+    assert_script_run "systemctl enable --now openqa-scheduler";
+    assert_script_run "systemctl status --no-pager openqa-scheduler";
```

Comment:
Here is how it cuold look like with an inented here-document:
```suggestion
    assert_script_run($_) foreach (split /\n/, <<~'EOF');
        sed -i -e 's/#.*method.*OpenID.*$/&\nmethod = Fake/' /etc/openqa/openqa.ini
        systemctl enable --now openqa-webui
        systemctl status --no-pager openqa-webui
        systemctl enable --now openqa-scheduler
        systemctl status --no-pager openqa-scheduler
        EOF
```

I'm fine with either way.

Reply:
I do find the here-doc a bit more readable. You see what's going on right away, except for the bit cryptic split above the actual shell code.